tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -15,6 +15,9 @@ jobs:
   - script: brew install coreutils ninja
     displayName: Install coreutils and ninja
 
+  - script: /bin/bash tools/ci_build/github/android/setup_gradle_wrapper.sh $(pwd)
```

Comment:
nit: the script itself seems simple enough to specify inline in the build definition. and then you can use $(Build.SourcesDirectory) directly.

Reply:
Yes, but I was thinking maybe macOS pipeline also need this, so keep this as a shell script to be easily added to other pipelines if necessary.